dlrover/python/diagnosis/common/constants.py

```diff
@@ -12,12 +12,27 @@
 # limitations under the License.
 
 
+class EnvConfigKey(object):
+    XPU_TIMER_PORT = "XPU_TIMER_PORT"
+
+
 class InferenceConfigKey(object):
     LOG_FILE = "log_file"
     ERRORS = "errors"
 
 
-class DiagnoseAction(object):
+class DiagnosisConstant(object):
+    MASTER_DIAGNOSIS_OBSERVING_INTERVAL_SECS = 180
+    AGENT_PERIODICALLY_DIAGNOSIS_INTERVAL_SECS = 60
+
+
+class DiagnosisDataType(object):
+    GENERIC = "GENERIC"
+    TRAINING_LOG = "TRAINING_LOG"
+    TRAINING_HANG_DETECTION = "TRAINING_HANG_DETECTION"
```

Comment:
"TRAINING_HANG_DETECTION" seems not to be the type name of data. Maybe stack data or something similar?

Reply:
Change to ```XPU_TIMER_METRIC```